.eslintignore

```diff
@@ -0,0 +1 @@
+luxon.js
```

Comment:
- [ ] In readme file, please update the live, it is working but not with your latest changes

Reply:
Hi @GzimAsani , 
Thank you kindly for bringing this to our attention! We have updated the live link and it is now reflecting the latest changes.